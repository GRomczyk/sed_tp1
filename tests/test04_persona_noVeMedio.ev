00:00:01:0 in_port1 [1.0,1.0]
00:00:10:0 info 1

