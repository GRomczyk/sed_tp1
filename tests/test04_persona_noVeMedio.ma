[Top]
%en este test esperamos que no vea el medio por tener tiempo sillon 0
% su p deberia quedar constante
components : P1@Persona

out : info_out1
in : in_port1 info

Link : in_port1 inMedios@P1
Link : info infoRequest@P1

Link : outInfo@P1 info_out1

[P1]
p : 0.5
influenciabilidad : 0.9
poder_convencimiento : 0.2
tiempo_sillon : 0.0
grado : 1
periodo_medio : 60000000
