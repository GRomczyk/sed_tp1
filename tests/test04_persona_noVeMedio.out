00:00:10:000:0 info_out1      0.50000
